Remove trigger prop from `Tooltip` (#3103)
vivid-planet/comet: docs/docs/7-migration-guide/migration-from-v7-to-v8.md

+ import { Tooltip } from "@comet/admin";
```

### Remove `trigger` prop from `Tooltip`

> The `trigger` prop is deprecated and must be removed. The combined `hover`/`focus` trigger is now the only supported behavior.

Reviewer: ```suggestion
The `trigger` prop has been removed. The combined `hover`/`focus` trigger is now the only supported behavior.
```

> The `trigger` prop is deprecated and must be removed. The combined `hover`/`focus` trigger will be the only supported behavior.

Reviewer 2: ```suggestion
The `trigger` prop is deprecated and must be removed. The combined `hover`/`focus` trigger is now the only supported behavior.
```
